scripts/wallet.py

```diff
@@ -55,6 +55,103 @@ def keygen(args):
     print('Mnemonic   :', mnemonic)
     print_key_info(keypair)
 
+
+def unlock(args):
+    acct = Account(args)
+    acct.client.unlock_request_for_withdrawal(acct.keypair)
+
+def withdraw(args):
+    acct = Account(args)
+    acct.client.withdraw_stake(acct.keypair)
+
+
+def lock(args):
+    acct = Account(args)
+    utxo_value = Decimal()
+    utxos = []
+    amount = int(args.amount * MLT_UNIT)
+
+    if amount < 0:
+        raise Exception('Sending a negative amount')
+    elif amount <  40000:
+        raise Exception('Minimum amount to stake is 40,000 MLT')
+
+    for (h, u) in acct.mlt_utxos():
+        utxos.append((h, u))
+        utxo_value += u.value
+        if utxo_value >= amount:
+            break
+    if utxo_value < amount:
+        raise Exception('Not enough funds')
+
+    fee = int(MLT_UNIT / 10) # TODO
+    if fee >= 2 ** 64:
```

Comment:
Does this check make sense? Isn't `MLT_UNIT` a constant

Reply:
`MLT_UNIT` is 1 MLT, MLT_UNIT / 10 is 0.1 MLT hardcoded fee. I have an update where you can specify the fee MLT amount on the command line for payments